src/Controllers/SchemaController.js

```diff
@@ -91,7 +91,7 @@ const requiredColumns = Object.freeze({
   _Role: ["name", "ACL"]
 });
 
-const systemClasses = Object.freeze(['_User', '_Installation', '_Role', '_Session', '_Product']);
+const systemClasses = Object.freeze(['_User', '_Installation', '_Role', '_Session', '_Product', '_PushStatus']);
```

Comment:
Yeah. At this point I think it's the best plan. _PushStatus in Parse.com is not a Parse Object, but in Parse Server it does seem to be a Parse Object, kinda. We don't want it to be changeable by clients though, maybe I should add a test for that. Making it visible in the dash and/or modifiable by master key seems fine though.

We definitely want to have the ability to store _PushStatus in a separate DB, so having it go through the database adapter is the right way to go.